wiki/People/Beatmap_Nominators/Rules/en.md

```diff
@@ -18,27 +18,20 @@ Sustained behaviour in conflict with these expectations is grounds for dismissal
-  - **Falling below the 90 day requirements with more than 60 days' worth of nominations will result in an activity warning.**
-  - When warned for activity, minimum activity requirements for your respective game mode(s) must be met over the course of one month.
-  - Failing to meet the required minimum when being warned for it prior may result in removal from the Beatmap Nominators.
-- Beatmap Nominators are expected to stay below a maximum number of nominations over any 90-day period, with each mode having its own nomination cap. Exceeding this amount will lead to a warning, followed by a removal if they fail to manage their activity accordingly.
-  - osu!, osu!catch, osu!mania: 75 unique nominations.
-  - osu!taiko: 100 unique nominations.
-- Beatmap Nominators returning on good terms are expected to reach their mode's minimum activity requirements in the first 30 days since their rejoin. Staying inactive during this period of time may result in a warning or a removal, which will be evaluated on a case-by-case basis.
-
-### Requirements
-
-Beatmap Nominator activity is primarily measured by nominations. Resetting beatmaps is taken into account but to a lesser extent, and it has no set ratio compared to nominations.
-
-Each mode has its own required minimum for monthly activity.
-
-| Game mode | Requirement |
-| :-- | :-- |
-| osu!, osu!taiko | 3 nominations |
-| osu!catch, osu!mania[^multi-key-mode] | 2 nominations |
-| Hybrid | 3 nominations, with at least 1 per proficient game mode |
+Beatmap Nominator activity is measured via nominations[^multi-key-mode], and is evaluated during regular BN evaluations, which happen around every 90 days.
+
+### Minimum activity requirements
+
+- Beatmap Nominators must nominate at least **6** beatmaps over a 90-day period.
+  - Falling below **4** nominations in a 90-day period will result in a removal.
+  - Falling below the **6** nomination requirement with **4** or more nominations will result in an activity warning.
+- Hybrid Beatmap Nominators must nominate at least **3** beatmaps per proficient game mode over a 90-day period.
+  - Falling below **2** nominations in a given game mode in a 90-day period will result in a removal from that game mode.
+  - Falling below the **3** nomination requirement in a given game mode with **2** or more nominations will result in an activity warning in that game mode.
+- When warned for activity, minimum activity requirements must be met over the course of one month.
+- Failing to meet the required minimum when being warned for it within a period of 1 year will result in a removal from the Beatmap Nominators.
+
+When a Beatmap Nominator's evaluation is created, and they still need to meet their minimum activity requirements, they will receive a reminder via the [NAT bot](https://osu.ppy.sh/users/6616586) and will be given **7** days to meet their quota.
```

Comment:
not enough details to understand how this part affects all of the warning rules above, or at least it's really confusing

Reply:
i don't really understand how this is confusing. when an eval is created and the BN hasn't met their activity, they're given a 7-day leeway to reach their activity minimum before we finalize a decision, and are reminded via the nat bot in the form of a DM.

Reply:
I can't tell if it means you have 7 days to produce a 90-day period where you meet the quota, or if you have 7 days to make up the difference between the quota and whatever you nominated in the 90 days leading up to the eval. and if you ultimately fail the check after 7 days, does your "activity warning" start at the time of the eval or 7 days past that?

your reply now makes it sound like this is all a roundabout way to write "97-day period" :thinking:

Reply:
> or if you have 7 days to make up the difference between the quota and whatever you nominated in the 90 days leading up to the eval

this

> your reply now makes it sound like this is all a roundabout way to write "97-day period"

ehhh, kinda? intention was 90 days with a 7-day grace period to catch up, given evals tend to last 14 days.